web/styles/settings.css

```diff
@@ -513,14 +513,20 @@ input[type="checkbox"] {
     vertical-align: top;
     height: auto !important;
     width: auto !important;
+    white-space: break-spaces !important;
 
+    overflow-wrap: anywhere;
     background-color: transparent;
     border-radius: 4px;
-    margin-top: 14px;
-    margin-left: 10px;
```

Comment:
I think this has been accidentally removed.

Reply:
Reverted.